File: ifthenelseif.vb
' ifthenelseif.vb
Module IfThenElseIf
    Sub Main()
        Dim x As Integer = 5
        If x = 5 Then
            Console.WriteLine("x is 5")
        ElseIf x < 10 Then
            Console.WriteLine("x is less than 10")
        End If
    End Sub
End Module

File: ifthenelseif.cpp
// ifthenelseif.cpp
#include <iostream>

int main() {
    int x = 5;
    if (x == 5) {
        std::cout << "x is 5" << std::endl;
    } else if (x < 10) {
        std::cout << "x is less than 10" << std::endl;
    }
    return 0;
}
